### ISSUE TITLE
[Feature Request] Dynamic Theme changing based on os-parameters

### ISSUE THREAD

**Issue by zDEFz** [revoltchat/revite#555]
### What do you want to see?

# Dynamic Theme changing based on os-parameters
- https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
- For Accessibility, find a way to also detect high-contrast-white and high-contrast-black themes.

### PWA

- [X] Yes, this feature request is specific to the PWA.

**Comment by Rexogamer** [comment]
some thoughts:
- this should be a third theme option (i.e. dark, light, match)
- enabled by default for new users? use detected scheme as default??

**Comment by Freeplayg** [comment]
For the contrast bit, there's https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
However, it still isn't supported in Firefox yet (they have a flag you can enable in about:config).
